addons/test_website/static/tests/tours/website_settings.js

```diff
@@ -0,0 +1,44 @@
+/** @odoo-module */
+
+import tour from "web_tour.tour";
+
+function selectWebsiteSetting(websiteName) {
+    return [{
+        content: "Open website switcher",
+        trigger: "input#website_id",
+    }, {
+        content: `Select ${websiteName} in the website switcher`,
+        trigger: `li:has(.dropdown-item:contains('${websiteName}'))`,
+    }];
+}
+
+tour.register("website_settings", {
+    test: true,
```

Comment:
The `/web` is already defined in the python call of this tour `self.start_tour("/web", 'website_settings', login="admin")`
But yes I can add it here too so that, if someone want to execute the test via the frontend it will work too :+1: